Description: Fix -version-info option to keep SOVERSION at 7.
 In 4.2.1-RC0 the libtool version-info was changed from 7:0:0 to 7:0:1,
 causing the SOVERSION to decrement from 7 to 6.
 SpatiaLite 4.2.1 only introduced symbols,
 for which a SOVERSION change is not required.
 .
 In 4.3.0-RC0 the libtool version-info was changed from 7:0:1 to 8:0:1,
 but the missing - in the version-info option caused the SOVERSION
 to decrement from 6 to 0.
Author: Bas Couwenberg <sebastic@debian.org>
Forwarded: https://groups.google.com/d/msg/spatialite-users/oAzONsvEK08/rjzErEKVsuUJ

--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -42,10 +42,10 @@ libspatialite_la_LDFLAGS = -version-info
 libspatialite_la_LIBADD += -lm
 else 
 if ANDROID
-libspatialite_la_LDFLAGS = version-info 8:0:1
+libspatialite_la_LDFLAGS = -version-info 8:0:1
 libspatialite_la_LIBADD += -ldl -lm
 else
-libspatialite_la_LDFLAGS = version-info 8:0:1
+libspatialite_la_LDFLAGS = -version-info 8:0:1
 libspatialite_la_LIBADD += -lpthread -ldl -lm
 endif
 endif
@@ -76,10 +76,10 @@ mod_spatialite_la_LDFLAGS = -module -avo
 mod_spatialite_la_LIBADD += -lm
 else 
 if ANDROID
-mod_spatialite_la_LDFLAGS = -module version-info 8:0:1
+mod_spatialite_la_LDFLAGS = -module -version-info 8:0:1
 mod_spatialite_la_LIBADD += -ldl -lm
 else
-mod_spatialite_la_LDFLAGS = -module version-info 8:0:1
+mod_spatialite_la_LDFLAGS = -module -version-info 8:0:1
 mod_spatialite_la_LIBADD += -lpthread -ldl -lm
 endif
 endif
